<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TiendaAgricola</class>
 <widget class="QMainWindow" name="TiendaAgricola">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1281</width>
      <height>671</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(220, 236, 220);</string>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>40</y>
       <width>551</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 72pt &quot;MS Shell Dlg 2&quot;;
font: 48pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Tienda agricola</string>
     </property>
    </widget>
    <widget class="QPushButton" name="boton_registrar_usuario">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>210</y>
       <width>281</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Registrar usuario</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bttn_realizar_compra">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>310</y>
       <width>281</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Realizar compra</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bttn_mostrar_facturas">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>410</y>
       <width>281</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Mostrar facturas</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bttn_salir">
     <property name="geometry">
      <rect>
       <x>1202</x>
       <y>617</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 0, 0);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Salir</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>630</y>
       <width>411</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Jefferson David Arteaga Bedoya - Programacion IV</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
